x/auth/client/rest/rest.go

```diff
@@ -0,0 +1,29 @@
+package rest
+
+import (
+	"github.com/gorilla/mux"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+)
+
+// RegisterRoutes registers the auth module REST routes.
+func RegisterRoutes(cliCtx context.CLIContext, r *mux.Router, storeName string) {
+	r.HandleFunc(
+		"/auth/accounts/{address}",
+		QueryAccountRequestHandlerFn(storeName, context.GetAccountDecoder(cliCtx.Codec), cliCtx),
+	).Methods("GET")
+
+	// TODO: Change path or mount under x/bank if possible
+	r.HandleFunc(
+		"/bank/balances/{address}",
```

Comment:
can we delete this endpoint? it's a dup of the one above

Reply:
Hmmm we should't _remove_ it but rather _move_ or _rename_ it. One endpoint returns the entire account and the other returns the balance.